lib/bap_disasm/bap_disasm_brancher.ml

```diff
@@ -40,9 +114,7 @@ let has_jumps =
     end)
 
 let rec dests_of_bil bil : dests =
-  if has_jumps bil then
-    Bil.fold_consts bil |> List.concat_map ~f:dests_of_stmt
-  else []
+  Bil.fold_consts bil |> List.concat_map ~f:dests_of_stmt
```

Comment:
do we really need to do this after the BIL transformation pipeline PR is merged?